hedera-node/src/main/java/com/hedera/services/txns/consensus/SubmitMessageTransitionLogic.java

```diff
@@ -109,12 +109,13 @@ public void doStateTransition() {
 					chunkInfo.getInitialTransactionID().getAccountID());
 			final var transactionIDLookup = aliasManager.lookUpPayerAccountID(
 					transactionBody.getTransactionID().getAccountID());
-			if (chunkAccountIDLookup.response() != OK) {
-				transactionContext.setStatus(chunkAccountIDLookup.response());
-			}
-			if (transactionIDLookup.response() != OK) {
-				transactionContext.setStatus(transactionIDLookup.response());
-			}
+			/* I dont think we need these checks as it is handled just aftet these checks */
+//			if (chunkAccountIDLookup.response() != OK) {
+//				transactionContext.setStatus(chunkAccountIDLookup.response());
+//			}
+//			if (transactionIDLookup.response() != OK) {
+//				transactionContext.setStatus(transactionIDLookup.response());
+//			}
```

Comment:
why are we removing these ?

Reply:
I was actually not sure why we didn't `return` after the err response for each.